mne/io/meas_info.py

```diff
@@ -2906,6 +2906,10 @@ def _ensure_infos_match(info1, info2, name, *, on_mismatch='raise'):
         raise ValueError(f'{name}.info[\'sfreq\'] must match')
     if set(info1['ch_names']) != set(info2['ch_names']):
         raise ValueError(f'{name}.info[\'ch_names\'] must match')
+    if info1['ch_names'] != info2['ch_names']:
+        msg = (f'{name}.info[\'ch_names\']: Channel order must match. Use '
+                'mne.match_channel_orders() to sort channels.')
```

Comment:
does it even work to call it as `mne.match_channel_orders()`? (see my earlier comment about adding it to `mne.__init__.py`)

Reply:
no that is not what I meant. What I meant was will it really work for users to do what is recommended in the error message: to say `from mne import match_channel_orders` (or the equivalent `import mne; mne.match_channel_orders`).  I think it will *not* work because the function isn't in `mne/__init__.py` (or at least it wasn't when I reviewed last).

aside: cross-referencing the documentation by using `:func:` is not helpful inside error messages, which are always shown in the terminal / never interpreted as reStructuredText (only docstrings, tutorials, and other webpages need such crossreferences).

Reply:
I've added it in 1a801a638373dbe53aee75b31a1ca96915ea1677